### ISSUE TITLE
Error on local cluster - Project ID must be specified in the configuration

### ISSUE THREAD

**Issue by abonillabeeche** [GoogleCloudPlatform/microservices-demo#647]
### Describe the bug 
While trying to deploy on a local cluster, two pods fail because they expect a Project ID.

### To Reproduce 
1. git clone repo
2. skaffold run --default-repo=jfrog.privaterepo.com/onlineboutique

The same error is visible when using images/manifests built via `make-release.sh`

### Logs  
```
 - deployment/currencyservice: container server terminated with exit code 1
    - pod/currencyservice-9ddc9bd4f-x5grr: container server terminated with exit code 1
      > [currencyservice-9ddc9bd4f-x5grr server] Profiler enabled.
      > [currencyservice-9ddc9bd4f-x5grr server] Tracing enabled.
      > [currencyservice-9ddc9bd4f-x5grr server] Debugger enabled.
      > [currencyservice-9ddc9bd4f-x5grr server] {"severity":"info","time":1638511322161,"pid":1,"hostname":"currencyservice-9ddc9bd4f-x5grr","name":"currencyservice-server","message":"Starting gRPC server on port 7000...","v":1}
      > [currencyservice-9ddc9bd4f-x5grr server] (node:1) [changeLevelName_deprecation] Warning: The changeLevelName option is deprecated and will be removed in v7. Use levelKey instead.
      > [currencyservice-9ddc9bd4f-x5grr server] (Use `node --trace-warnings ...` to show where the warning was created)
      > [currencyservice-9ddc9bd4f-x5grr server] {"severity":"info","time":1638511322665,"pid":1,"hostname":"currencyservice-9ddc9bd4f-x5grr","name":"currencyservice-server","message":"CurrencyService gRPC server started on port 7000","v":1}
      > [currencyservice-9ddc9bd4f-x5grr server] /usr/src/app/node_modules/@google-cloud/profiler/build/src/index.js:120
      > [currencyservice-9ddc9bd4f-x5grr server]         throw new Error('Project ID must be specified in the configuration');
      > [currencyservice-9ddc9bd4f-x5grr server]               ^
      > [currencyservice-9ddc9bd4f-x5grr server] Error: Project ID must be specified in the configuration
      > [currencyservice-9ddc9bd4f-x5grr server]     at initConfigMetadata (/usr/src/app/node_modules/@google-cloud/profiler/build/src/index.js:120:15)
      > [currencyservice-9ddc9bd4f-x5grr server]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
      > [currencyservice-9ddc9bd4f-x5grr server]     at async createProfiler (/usr/src/app/node_modules/@google-cloud/profiler/build/src/index.js:158:26)
      > [currencyservice-9ddc9bd4f-x5grr server]     at async Object.start (/usr/src/app/node_modules/@google-cloud/profiler/build/src/index.js:182:22)
 - deployment/currencyservice failed. Error: container server terminated with exit code 1.
```
The same error is on both currencyservice and paymentservice.

### Environment 
MacOS Monterrey
Rancher Desktop && Kubernetes 1.19

**Comment by bourgeoisor** [comment]
The local K8s deployment instructions are here: https://github.com/GoogleCloudPlatform/microservices-demo/blob/master/docs/development-guide.md#option-2---local-cluster

From the logs you've posted, it sounds like the GCP Operations tools are attempting to initiate, which shouldn't be the case for local deployments (since there's no GCP projects to initiate those tools in).

Marking this as a bug, we'll investigate.

**Comment by frozenprocess** [comment]
Same error using release `v0.3.2`

@abonillabeeche  try release `v0.3.1` it worked for me.

**Comment by deeeed** [comment]
Same issue here, worked n v0.3.1. Thanks

**Comment by deeeed** [comment]
Same error for both currency and payment services.
Tried v0.3.1 which solved currency but payment service still didn't work. 
Couldn't start the app using skaffold dev.
Is there a way to set the PROJECT_ID in the minikube env?

**Comment by yogeek** [comment]
Same error, I ended up by patching (with kustomize) the deployments to disable PROFILING and DEBUGING :
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: paymentservice
spec:
  template:
    spec:
      containers:
      - name: server
        env:
        # - name: DISABLE_TRACING
        #   value: "1"
        - name: DISABLE_PROFILER
          value: "1"
        - name: DISABLE_DEBUGGER
          value: "1"
```

**Comment by ahrarmonsur** [comment]
Duplicated by https://github.com/GoogleCloudPlatform/microservices-demo/issues/663 for which a fix is in PR:
https://github.com/GoogleCloudPlatform/microservices-demo/pull/665

**Comment by askmeegs** [comment]
Fixed by PR #665 - these default manifest changes (removing GCP integrations by default) will land in the next release.
